modules/reference/pages/rpk/rpk-security/rpk-security-user-update.adoc

```diff
@@ -4,6 +4,8 @@
 
 Update SASL user credentials
 
+CAUTION: The default value for the `--mechanism` flag is `SCRAM-SHA-256`. Specify the flag when using a different mechanism to avoid unexpected changes.
```

Comment:
```suggestion
CAUTION: The default value for the `--mechanism` flag is `SCRAM-SHA-256`. Set the flag when using a different mechanism to avoid unexpected changes.
```